cocos/2d/components/ui-opacity.ts

```diff
@@ -133,8 +134,32 @@ export class UIOpacity extends Component {
     @serializable
     protected _opacity = 255;
 
+    protected _parentAdded (node: Node): void {
+        const parent: Node |null = node.getParent();
+        if (parent == null) {
+            return;
+        }
```

Comment:
Why should check it as it is a callback of CHILD_ADDED.

Reply:
Because getParent may be null

Reply:
It is called when a child is added, then why parent will be null?